play.py

```diff
@@ -1,24 +1,17 @@
-#!/usr/bin/env python
```

Comment:
sorry to be a pain but can you keep this comment? This is useful if you want to execute directly on unix systems.